test/instrumentation/modules/express/set-framework.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const agent = require('../../../..').start({
+  captureExceptions: true,
+  metricsInterval: 0
+})
+
+let asserts = 0
+
+agent.setFramework = function ({ name, version, overwrite }) {
+  asserts++
+  assert.strictEqual(name, 'express')
+  assert.strictEqual(version, require('express/package').version)
+  assert.strictEqual(overwrite, false)
+}
+
+const assert = require('assert')
+
+require('express')
+
+assert.strictEqual(asserts, 1)
```

Comment:
Does it work to just mix assert-only tests like this with our tape-based stuff?

Reply:
It should work ok as we're just spawning `node <test-file>` from within `test/test.js`. We use the `assert` module in a few other tests as well.

The reason why I went with `assert` here was to reduce the number of modules being required to ensure there were no unintended side-effects, which was also why I added it as a separate file (I tried to integrate it into the main test file for each framework first, but that had major drawbacks).